src/app/pages/administration/administration.html

```diff
@@ -1,9 +1,4 @@
 <div class="container-fluid">
-  <div class="row">
-    <div class="col-sm-12">
-      <h1>CHPL Administration</h1>
```

Comment:
Removing this `h1` means that every other page under the Administration section needs to be changed, as none of them will have an `h1` in their tree anymore, which means they're not correct in a 508 sense, nor are they consistent in a page-by-page way